RELEASENOTES.md

```diff
@@ -4,6 +4,13 @@ This file contains highlights and announcements covering all components.
 For more details see `CHANGELOG.md` files maintained in the root source
 directory of each individual package.
 
+## 1.12.0
+
+Release details: [1.12.0](https://github.com/CodeBlanchOrg/opentelemetry-dotnet/releases/tag/core-1.12.0)
```

Comment:
Note: When using automation this will be a dead link initially. It will become live when the release is published at the end of the process.